src/appsettings.json

```diff
@@ -0,0 +1,6 @@
+﻿{
+  "TenantId": "[Put your Directory (tenant) ID here]",
+  "ClientId": "[Put your Application (client) ID here]",
+  "SubscriptionId": "[Put your Subscription ID here]", // Can be found in the Azure Portal here: https://portal.azure.com/#blade/Microsoft_Azure_Billing/SubscriptionsBlade
```

Comment:
I think the comment has to go in the string; Vanilla JSON doesn't support comments.

Reply:
fixed